cmd/server/upsClient.go

```diff
@@ -15,8 +15,34 @@ type upsClient struct {
 
 const BaseUrl = "http://localhost:8080/rest/applications"
 
+// Delete the variant with the given google key
+func (client *upsClient) deleteAndroidVariant(key string) bool {
+	variant := client.hasAndroidVariant(key)
+	if variant != nil {
+		log.Printf("Deleting variant with id `%s`", variant.VariantID)
+
+		url := fmt.Sprintf("%s/%s/adm/%s", BaseUrl, client.config.ApplicationId, variant.VariantID)
+		log.Printf("UPS request", url)
+
+		req, err := http.NewRequest("DELETE", url, nil)
```

Comment:
does the http client lib not have "constants" for the different methods? instead of using string ?

Reply:
It does have specialized functions for GET and POST, i'll have a look if there are constants for the other methods that i can use.